Source/GameView/GameViewModels/GameDataViewModel.swift

```diff
@@ -39,13 +32,18 @@ public class GameDataViewModel {
    
    - Returns: A tuple of the necessary details for the view to display.
    */
-  public func getGameDetails(at index: Int) -> (name: String, gamePrice: String) {
+  public func getGameDetails(at index: Int) -> (gameName: String, gamePrice: String) {
+    
+    let game = model.getGameAt(index: index)
```

Comment:
Minor suggestion here for the future, when you're writing a function try and make it sound as much as a readable sentence as possible. In this case you're saying get game at index index.

if we instead have `getGameAt(_ index: Int)` as our function signature then we can call it with:
`getGameAt(index)` which reads more correctly

Just useful for the future